hawkbit-dmf/hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpMessageHandlerService.java

```diff
@@ -355,6 +355,11 @@ private void updateActionStatus(final Message message) {
         final ActionStatusCreate actionStatus = entityFactory.actionStatus().create(action.getId()).status(status)
                 .messages(messages);
 
+        actionUpdateStatus.getCode().ifPresent(code -> {
+            actionStatus.code(code);
+            actionStatus.message("Device reported code: " + code);
```

Comment:
Naming the field "code" is fine for me. How about using "status code" within the message to make it more tangible: "Device reported status code: ..."

Reply:
adapted